src/dynamics/senh/dyn_comp.F90

```diff
@@ -114,7 +114,10 @@ subroutine dyn_readnl(NLFileName)
   use fvm_mod,        only: fvm_get_test_type
   use native_mapping, only: native_mapping_readnl
 #endif
-  use dimensions_mod_cam, only: qsize, qsize_d, ntrac, ntrac_d, npsq, ne, npart, lcp_moist
+#ifndef HOMME_WITHOUT_PIOLIBRARY
```

Comment:
Where is the CPP variable being defined?

Reply:
This was from the E3SM code.  meshopen isn't used here anymore so I got rid of the CPP variable.